src/Fantomas.Tests/TupleTests.fs

```diff
@@ -268,3 +268,61 @@ let y =
          sprintf "(%s)" args),
     namesWithIndices
 """
+
+[<Test>]
+let ``comment on first tuple argument is preserved`` () =
+    formatSourceString
+        false
+        """
+let func (a, b) = a + b
+
+func(
+        // abc
+        0,
+        1
+)
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+let func (a, b) = a + b
+
+func (
+    // abc
+    0,
+    1
+)
+"""
+
+[<Test>]
+let ``comment trivias on tuple arguments are preserved`` () =
+    formatSourceString
+        false
+        """
+let func (a, b) = a + b
+
+func(
+        // abc
+        0, // def
+        // ghi
+        1 // jkl
+        // mno
+)
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+let func (a, b) = a + b
+
+func (
+    // abc
+    0,  // def
```

Comment:
Good spot, this can be solved by changing `long` to
```fsharp
            let long =
                genExpr astContext e
                +> sepSpace
                +> sepOpenTFor lpr
                +> indent
                +> sepNln
                +> (col (sepCommaFixed +> sepNln) args (genExprLong astContext)
                    |> genTupleTrivia)
                +> unindent
                +> sepNln
                +> sepCloseTFor rpr
```

Notice that sepCommaFixed does not exist yet.
You can add it in Context (right under sepComma):
```fsharp
let internal sepCommaFixed = str ","
```

The `Fixed` convention means, give me the value without checking the configuration if there should be a space.